### PR TITLE
initial proposal

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

**Comment by zuazo** [comment]
Wow! I did not know we can do this. It's cool! 😃 

Only one question: some cookbooks are missing, is that intentional?

- [`bsdcpio`](https://github.com/sous-chefs/bsdcpio)
- [`nano`](https://github.com/sous-chefs/nano)
- [`unbound`](https://github.com/sous-chefs/unbound)

**Comment by josephholsten** [comment]
Unintentional, I guess I can put myself down for them.

File: recipes/default.rb [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+  members: %w(
+    damacus
+    josephholsten
+    tas50
+  )
+
+# Cookbook teams
+
+sc_team 'apache2',
+  members: %w(svanzoest)
```
Comment (shortdudey123) [inline recipes/default.rb]: should we make all of these multi-line? then adding another person in the future will be a cleaner diff

**Comment by josephholsten** [comment]
closing in favor of https://github.com/sous-chefs/terraform-github-org

**Pull request closed without merging**